controllers/news.controllers.js

```diff
@@ -12,3 +16,11 @@ exports.getTopics = (req, res, next) => {
     res.status(200).send({ topics });
   });
 };
+
+exports.getArticleById = (req, res, next) => {
+  selectArticleById(req.params.article_id)
```

Comment:
I think it would clearer here if you accessed the article_id from the req.params before passing it to the function 💭